### PR TITLE
Fix `url_rewrite_program`

### PR DESCRIPTION
**Pull request by SourceDoctor**
moves url_rewrite_program into Header, cause it's only once useable.
Also fixes some squid Configfile grammars and styles

### CODE REVIEW

**Comment by SourceDoctor** [comment]
Hi, something new about state for merging this fix?

**Comment by SourceDoctor** [comment]
something wrong with this merge Request?

File: templates/squid.conf.header.erb [hunk 1]
```diff
@@ -61,3 +61,10 @@ httpd_suppress_version_string <%= @httpd_suppress_version_string?'on':'off' %>
 <% unless @forwarded_for.nil? -%>
 forwarded_for                 <%= @forwarded_for?'on':'off' %>
 <% end -%>
+
+<% unless @url_rewrite_program.nil? -%>
```
Comment (alexjfisher) [inline templates/squid.conf.header.erb]: or just `<% if @url_rewrite_program -%>`?

**Comment by SourceDoctor** [comment]
@alexjfisher
is there something i've missed? i changed only from "unless" to "if" now 2/3 of tests are failing?

**Comment by alexjfisher** [comment]
@SourceDoctor It's not you, just the passage of time.  Since you first opened this PR, there's been new releases of beaker-puppet.  One of these broke some of our tests.  But since https://github.com/puppetlabs/beaker-puppet/pull/73 was merged, at least it's easy to figure out what needs fixing up. :)

https://github.com/voxpupuli/puppet-squid/blob/master/spec/acceptance/class_spec.rb#L5

**Comment by alexjfisher** [comment]
This should be fixed in https://github.com/voxpupuli/puppet-squid/pull/111

**Comment by alexjfisher** [comment]
Fix has been merged, so hopefully this PR will pass now. (travis tests the result of merging the PR branch to master, not the branch itself).

**Comment by SourceDoctor** [comment]
ok so? :)

**Comment by SourceDoctor** [comment]
oh, my fault.
Now it's like it should be :-)

**Comment by alexjfisher** [comment]
The email address you've used isn't registered on your github profile, (meaning the commit doesn't link back to your github id).  It also differs from the one you've used on your other commits. Is this intentional?

**Comment by SourceDoctor** [comment]
i had a typo, sry.

**Comment by alexjfisher** [comment]
Merged!  Thanks.

You might want to add the `googlemail.com` email variant to your github profile so that your previous commits also link to your account.

**Pull request merged**